jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/HttpConnectionFactory.java

```diff
@@ -57,45 +51,26 @@ public HttpConfiguration getHttpConfiguration()
         return _config;
     }
 
-    /**
-     * @deprecated use {@link HttpConfiguration#getComplianceViolationListeners()} instead to know if there
-     * are any {@link ComplianceViolation.Listener} to notify.  this method will be removed in Jetty 12.1.0
-     */
-    @Deprecated(since = "12.0.6", forRemoval = true)
-    public boolean isRecordHttpComplianceViolations()
-    {
-        return !_config.getComplianceViolationListeners().isEmpty();
-    }
-
-    /**
-     * Does nothing.
-     * @deprecated use {@link HttpConfiguration#addComplianceViolationListener(ComplianceViolation.Listener)} instead.
-     * this method will be removed in Jetty 12.1.0
-     */
-    @Deprecated(since = "12.0.6", forRemoval = true)
-    public void setRecordHttpComplianceViolations(boolean recordHttpComplianceViolations)
-    {
-        _config.addComplianceViolationListener(new ComplianceViolation.LoggingListener());
-    }
-
     public boolean isUseInputDirectByteBuffers()
     {
-        return _useInputDirectByteBuffers;
+        return _config.isUseInputDirectByteBuffers();
```

Comment:
Not here, but now in the constructor, the calls to `setUse*Buffers` are now redundant.